src/app/router/Welcome/Welcome.tsx

```diff
@@ -2,6 +2,7 @@ import { useEffect, useState } from "react";
 import { HashRouter as Router, useRoutes, useLocation } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import i18n from "~/i18n/i18nConfig";
+import {} from "~/i18n/namespaces";
```

Comment:
Looking into this, I think it’s better to delete so that default will beescapedValue: true. The docs default to using it because [React escapes HTML by default](https://stackoverflow.com/questions/33644499/what-does-it-mean-when-they-say-react-is-xss-protected). However, [as noted here](https://stackoverflow.com/questions/33644499/what-does-it-mean-when-they-say-react-is-xss-protected#comment92875465_33644981), there are other ways to inject malicious code that React does not protect against. Since we’re using strings in a variety of ways and since it doesn’t change the behavior I don’t see a reason to not escape by default, unless there’s some performance difference or something I’m missing.